engine/render/src/render/font_renderer_default.cpp

```diff
@@ -343,10 +347,10 @@ namespace dmRender
                         GlyphVertex& v5_layer_face = vertices[face_index + 4];
                         GlyphVertex& v6_layer_face = vertices[face_index + 5];
 
-                        (Vector4&) v1_layer_face.m_Position = te.m_Transform * Vector4(x + left_bearing, y - descent, 0, 1);
-                        (Vector4&) v2_layer_face.m_Position = te.m_Transform * Vector4(x + left_bearing, y + ascent, 0, 1);
-                        (Vector4&) v3_layer_face.m_Position = te.m_Transform * Vector4(x + left_bearing + width, y - descent, 0, 1);
-                        (Vector4&) v6_layer_face.m_Position = te.m_Transform * Vector4(x + left_bearing + width, y + ascent, 0, 1);
+                        (Vector4&) v1_layer_face.m_Position = te.m_Transform * Vector4(x + f_left_bearing, y - f_descent, 0, 1);
+                        (Vector4&) v2_layer_face.m_Position = te.m_Transform * Vector4(x + f_left_bearing, y + f_ascent, 0, 1);
+                        (Vector4&) v3_layer_face.m_Position = te.m_Transform * Vector4(x + f_left_bearing + f_width, y - f_descent, 0, 1);
+                        (Vector4&) v6_layer_face.m_Position = te.m_Transform * Vector4(x + f_left_bearing + f_width, y + f_ascent, 0, 1);
```

Comment:
Use the floating point values when calculating the positions.
This is especially important when considering a glyph that should "affect" the appearance of the previous glyph.
If the values are clamped, they start to misalign.